stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +507,42 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    fun setTextTitlePadding(@DimenRes size: Int) {
```

Comment:
It would be better to pass resolved dimension here, not a raw resource to let users hardcode the value. Same comment for the methods below.

Reply:
Fixed